pkg/utils/validation/admissionplugins/admissionplugins_test.go

```diff
@@ -113,5 +114,176 @@ var _ = Describe("admissionplugins", func() {
 				"Detail": Equal("forbidden admission plugin was specified - do not use plugins from the following list: [SecurityContextDeny]"),
 			})))),
 		)
+
+		Describe("validate PodSecurity admissionPlugin config", func() {
+			test := func(kubernetesVersion string, v1alpha1, v1beta1, v1 bool) {
+				if v1alpha1 {
+					It("should allow v1alpha1 configuration", func() {
```

Comment:
This `if else` structure is confusing, instead you can make three conditions:
1. v1alpha1, v1beta1 and v1 all are true.
2. only v1alpha1 and v1beta1 are true.
3. only v1alpha1 is true.

This will increase the overall readability of the code.

Reply:
Wouldn't it increase duplication? For eg, "should allow v1alpha1 configuration" should be tested for all three conditions.